content/release-notes/1.50.2.md

```diff
@@ -0,0 +1,16 @@
+---
+date: 2021-09-07T17:00:00-00:00
+linktitle: "1.50.2"
+title: "1.50.2"
+kubernetes: "1.19, 1.20, and 1.21"
+---
+
+{{<fixes>}}
+* Fixed a bug that caused KOTS to fail to detect [private images](/vendor/packaging/private-images/#replicated-private-registry) when their domains are inaccessible during online installs.
+* Fixed a bug that caused the [set config](/kots-cli/set/config/) command to fail if an appliction config has a required password value that was already set.
+* Fixed a bug that caused the [LocalImageName](/reference/template-functions/config-context/#localimagename) function to not require private images when `proxyPublicImages` is set.
+* Fixed a bug that could cause the secret used to pull [private images](/vendor/packaging/private-images/) to be removed during a [KOTS Snapshot](/kotsadm/snapshots/overview/) restore when multiple applications are installed.
+* Fixed a bug that could cause [Helm render](/vendor/helm/helm-processing/) on a chart deployed via the API to not be supported by the target Kubernetes cluster.
+* Fixed a bug where [private images](/vendor/packaging/private-images/) would fail to kustomize when the [experimental Helm rendering workflow beta](/release-notes/1.47.0/) for Helm charts was in use.
+* Fixed a bug where [private images](/vendor/packaging/private-images/) would fail to deploy when both existing [Helm Charts](/vendor/helm/using-helm-charts/) and the [experimental Helm rendering workflow beta](/release-notes/1.47.0/) Helm charts was in use. 
```

Comment:
```suggestion
* Fixed a bug where [private images](/vendor/packaging/private-images/) would fail to deploy when both existing [Helm Charts](/vendor/helm/using-helm-charts/) and the [experimental Helm rendering workflow beta](/release-notes/1.47.0/) Helm charts were in use. 
```